test/test_case_test.dart

```diff
@@ -11,6 +11,18 @@ void main() {
       expect(result.asJson, sampleForNewTestCase);
     });
 
+    test('#delete', () async {
+      stubTestRailConfig({});
+      final result = await TestCase.delete(1);
+      expect(result.asJson, {});
```

Comment:
Ah I see here, so instead of `TestCase.delete` returning an empty `SoftDeleteTestCase` could we instead return `null` ?